### PR TITLE
test: fix test on host with ipv6 disabled

### PR DESCRIPTION
**Pull request by godsarmy**
#### Description

Fix test failure on a host with ipv6 disabled

#### Checklist

 - [x] I have updated tests where applicable.
 - [ ] I have added an entry to the CHANGELOG.

### CODE REVIEW

File: test/mitmproxy/net/test_local_ip.py [hunk 1]
```diff
@@ -9,6 +10,7 @@ def test_get_local_ip():
     local_ip.get_local_ip("invalid!")
 
 
+@skip_no_ipv6
 def test_get_local_ip6():
     # should never error, but may return None depending on the host OS configuration.
```
Comment (mhils) [inline test/mitmproxy/net/test_local_ip.py:15]: The test is really here to ensure that we don't error, even in environments without IPv6. What error are you getting?

Comment (godsarmy) [reply test/mitmproxy/net/test_local_ip.py:15]: On a ipv6 diabled host, the error message is like

"OSError: [Errno 97] Address family not supporterd by protocol"

I think skip_no_ipv6 is used to skip the test when detecting ipv6 is diabled on the host

Comment (mhils) [reply test/mitmproxy/net/test_local_ip.py:15]: The test is correct, the underlying method needs to return None instead of raising.

Comment (godsarmy) [reply test/mitmproxy/net/test_local_ip.py:15]: so do you mean the code itself is wrong? If so, i can fix the code.

Comment (mhils) [reply test/mitmproxy/net/test_local_ip.py:15]: Yes, correct. Thanks!

**Pull request closed without merging**